### PR TITLE
fix ctrl-v paste doesn't work in command line mode

### PR DESCRIPTION
**Pull request by haifengkao**
ctrl-v (and cmd-v) doesn't paste anything in command line mode

### How this PR fixes the problem?

Use electron's default paste command to paste from clipboard

### Check lists (check `x` in `[ ]` of list items)

- [x] Coding style (if any code was modified)
- [x] Confirmed
  - OS: {OSX 10.11}
  - `nvim` version: {0.1.5}

### CODE REVIEW

**Comment by haifengkao** [comment]
This pull request will have troubles printing control characters.
The better way is to use ctrl-r ".

File: renderer/nyaovim-app.ts [hunk 1]
```diff
@@ -253,6 +253,9 @@ Polymer({
                         // insert mode
                         // gp will move cursor to the last of pasted content
                         command = '<esc>"+gpi';
+                    } else if (ch === 'c') {
+                         // command line mode
+                        command = '<c-r>+';
```
Comment (rhysd) [inline renderer/nyaovim-app.ts]: `<C-r>+` is also available in insert mode.  Should we also use it in the mode?

```javascript
} else if (ch === 'c' || ch === 'i') {
    command = '<c-r>+';
}
```

**Comment by haifengkao** [comment]
`<c-r>"` in insert mode will break the format
Same behavior for neovim and macvim.

**Comment by rhysd** [comment]
I think it's normal behavior for Vim but it's also fact that it's inconvenient.  OK, I'll introduce your implementation here.  Thank you.

**Pull request merged**